### PR TITLE
Added login or user creation onboarding script for SQL Insights

### PR DESCRIPTION
**Pull request by avinash-nigam**
Added login or user creation onboarding script and its corresponding readme file for SQL Insights solution

## PR Checklist

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

### If adding or updating templates:
* [ ] post a screenshot of templates and/or gallery changes
* [ ] ensure your template has a corresponding gallery entry in the gallery folder
* [ ] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [ ] ensure all steps have meaningful names
* [ ] ensure all parameters and grid columns have display names set so they can be localized
* [ ] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [ ] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [ ] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [ ] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/Workloads/SQL/SQL Insights Onboarding Scripts/Permissions_LoginUser_Account_Creation.ps1 [hunk 1]
```diff
@@ -0,0 +1,467 @@
...
+$loginname = 'telegraf'
+
+# Prefix for login password's secret name to be stored in keyvault
+$loginpasswordsecretnameprefix = 'telegraflogin-'
+
+# Name of the user to be created at a database level
+$username = 'telegrafuser'
+
+
+<#
```
Comment (vutran01) [inline Workbooks/Workloads/SQL/SQL Insights Onboarding Scripts/Permissions_LoginUser_Account_Creation.ps1:187]: should we just delete this comment?

File: Workbooks/Workloads/SQL/SQL Insights Onboarding Scripts/Permissions_LoginUser_Account_Creation.ps1 [hunk 2]
```diff
@@ -0,0 +1,467 @@
...
+$ConfigConnectionStringList = New-Object System.Collections.Generic.List[System.Object]
+$server_list = @{}
+$success_server_count = 0
+$failure_server_count = 0
+$partial_server_count = 0
+$success_database_count = 0
+$failure_database_count = 0
+[bool] $isDatabaseListDefined = 0
+
+# TODO: Do we want to support this?
```
Comment (vutran01) [inline Workbooks/Workloads/SQL/SQL Insights Onboarding Scripts/Permissions_LoginUser_Account_Creation.ps1:208]: have we resolve this todo?

Comment (avinash-nigam) [reply Workbooks/Workloads/SQL/SQL Insights Onboarding Scripts/Permissions_LoginUser_Account_Creation.ps1:208]: This was an open ended question and I guess it is still open, so for now I can remove this comment, and we can revisit this later when we have a better clarity on it.

File: Workbooks/Workloads/SQL/SQL Insights Onboarding Scripts/Permissions_LoginUser_Account_Creation.ps1 (hunk 1 above)
Comment (avinash-nigam) [reply Workbooks/Workloads/SQL/SQL Insights Onboarding Scripts/Permissions_LoginUser_Account_Creation.ps1:187]: sure, I'll take it off

File: Workbooks/Workloads/SQL/SQL Insights Onboarding Scripts/Permissions_LoginUser_Account_Creation.ps1 (hunk 2 above)
Comment (vutran01) [reply Workbooks/Workloads/SQL/SQL Insights Onboarding Scripts/Permissions_LoginUser_Account_Creation.ps1:208]: same with the todo a down below. customers will be running this so don't want them to see bunch of todo's

Comment (avinash-nigam) [reply Workbooks/Workloads/SQL/SQL Insights Onboarding Scripts/Permissions_LoginUser_Account_Creation.ps1:208]: yep makes sense, saw that and took it out too

File: Workbooks/Workloads/SQL/SQL Insights Onboarding Scripts/Permissions_LoginUser_Account_Creation.ps1 (hunk 1 above)
Comment (avinash-nigam) [reply Workbooks/Workloads/SQL/SQL Insights Onboarding Scripts/Permissions_LoginUser_Account_Creation.ps1]: done!

File: Workbooks/Workloads/SQL/SQL Insights Onboarding Scripts/Permissions_LoginUser_Account_Creation.ps1 (hunk 2 above)
Comment (avinash-nigam) [reply Workbooks/Workloads/SQL/SQL Insights Onboarding Scripts/Permissions_LoginUser_Account_Creation.ps1]: done!

**Review by dougbrad — APPROVED** [review]

**Pull request merged**